extern/acts/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ message( STATUS "Building Acts as part of the TRACCC project" )
 
 # Declare where to get Acts from.
 set( TRACCC_ACTS_SOURCE
-   "URL;https://github.com/krasznaa/acts/archive/refs/tags/v10.0.1.tar.gz;URL_MD5;fcd2f2f7b5810e34b8faa12634043be3"
+   "GIT_REPOSITORY;https://github.com/guilhermeAlmeida1/acts.git;GIT_TAG;patch-1"
```

Comment:
Let me just point it out here as well, we'll of course need a new tag of the main Acts repo before this PR would be accepted.

Even in the worst case, since your fix is already included in the `main` branch of the main Acts repository, this setting would rather have to be:

```
"GIT_REPOSITORY;https://github.com/acts-project/acts.git;GIT_TAG;origin/main"
```